### PR TITLE
Represent architecture

### PR DESCRIPTION
**Pull request by xiayy860612**

### CODE REVIEW

File: doc/architechture.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Architechture
+
+```plantuml
+@startuml "Workflow"
```
Comment (gyfpython) [inline doc/architechture.md:4]: Lack of response data flow.
BlogContents in StaticResources will be store into Elasticsearch? Lock of this flow.
Where to store WebResources, file system?
Nginx as a Loadbalance or a request distributor(the configuration of this two strategy are different)

Comment (xiayy860612) [reply doc/architechture.md:4]: Currently No response flow in architechture cause this isn't a request/response work flow, 
it's only user activity and server activity flow in deployed components.

folder in puml means use file system.
all static resources will be stored in file system.
 
We will store keywords of content elasticsearch for blog content full-text search to get related blogs.

@gyfpython I will update the name for Nginx usage, 
it's used for reverse-proxy include loadbalance and static resources cache.

**Review by gyfpython — APPROVED** [review]
Reviewed

**Comment by xiayy860612** [comment]
fix #3

**Pull request merged**